modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/session.js

```diff
@@ -328,46 +328,42 @@ AUI.add(
 									elapsed = timeOffset;
 								}
 
-								var extend = false;
+								var extend = instance.get('autoExtend');
 
-								var expirationMoment = elapsed == sessionLength;
-								var warningMoment = elapsed == warningTime;
+								var expirationMoment = false;
+								var warningMoment = false;
 
 								var hasExpired = elapsed >= sessionLength;
 								var hasWarned = elapsed >= warningTime;
 
 								var updateSessionState = true;
 
 								if (hasWarned) {
-									if (warningMoment || expirationMoment) {
-										if (timestamp == 'expired') {
-											expirationMoment = true;
-											hasExpired = true;
-										}
-										else if (instance.get('autoExtend')) {
-											expirationMoment = false;
-											extend = true;
-											hasExpired = false;
-											hasWarned = false;
-											warningMoment = false;
-										}
-										else if (timeOffset < warningTime) {
-											hasWarned = false;
-											updateSessionState = false;
-										}
+									if (timestamp == 'expired') {
+										expirationMoment = true;
+										hasExpired = true;
 									}
 
 									if (updateSessionState) {
```

Comment:
This is always set to ```true``` (See https://github.com/jonmak08/liferay-portal/pull/1662/commits/96f12419597d08640ee2c4bc109b58d79be730af#diff-1ad2f307ca967b71dac6d9fa38581166L339).  What is the purpose in having this line  now that is will always be set to ```true```?